gradle-plugins/build.gradle.kts

```diff
@@ -53,9 +54,31 @@ gradlePlugin {
   plugins {
     get("io.opentelemetry.instrumentation.muzzle-generation").apply {
       displayName = "Muzzle safety net generation"
+      description = "https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/contributing/muzzle.md"
     }
     get("io.opentelemetry.instrumentation.muzzle-check").apply {
       displayName = "Checks instrumented libraries against muzzle safety net"
+      description = "https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/contributing/muzzle.md"
     }
   }
+}
+
+nexusPublishing {
+  packageGroup.set("io.opentelemetry")
+
+  repositories {
+    sonatype {
+      username.set(System.getenv("SONATYPE_USER"))
+      password.set(System.getenv("SONATYPE_KEY"))
+    }
+  }
+
+  connectTimeout.set(Duration.ofMinutes(5))
+  clientTimeout.set(Duration.ofMinutes(5))
+}
+
+tasks {
+  publishPlugins {
+    enabled = !version.toString().contains("SNAPSHOT")
```

Comment:
should `version` above be changed back to SNAPSHOT for now?

Reply:
I really want to publish this to Gradle Portal. The whole idea of the previous PR was to achieve that.

Reply:
is `enabled` setting here needed, since we only run `publishPlugins` from manual workflow? is it just for safety?

Reply:
Mostly for safety, yes. I tried to disable `publishToSonatype` as well, but it gradle did not find such task :( Probably some evaluation order or rule magic :(